### PR TITLE
[2.3 branch] remove wget in df_stats example

### PR DESCRIPTION
**Pull request by chesterxgchen**
### Description
remove wget to fix broken example. This is the same as https://github.com/NVIDIA/NVFlare/pull/1878 for dev branch

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Review by IsaacYangSLA — APPROVED** [review]
LGTM.  Thanks for the PR.

File: examples/advanced/federated-statistics/df_stats/utils/prepare_data.py [hunk 1]
```diff
@@ -60,9 +59,15 @@ def prepare_data(data_root_dir: str):
             print(f"\nremove existing data at {dest}")
             shutil.rmtree(dest, ignore_errors=True)
 
-        print(f"\nwget download to {dest}")
+        print(f"\ndownload to {dest}")
         url = client_data_urls[client]
-        response = wget.download(url, dest)
+        import requests
```
Comment (yhwen) [inline examples/advanced/federated-statistics/df_stats/utils/prepare_data.py:64]: Move this "import requests" to the top?

File: examples/advanced/federated-statistics/df_stats/utils/prepare_data.py [hunk 2]
```diff
@@ -13,11 +13,10 @@
 # limitations under the License.
 
 import argparse
+import csv
```
Comment (YuanTingHsieh) [inline examples/advanced/federated-statistics/df_stats/utils/prepare_data.py:16]: do we need to add this in "examples/advanced/federated-statistics/df_stats/requirements.txt"
or it is builtin?

**Pull request merged**